### PR TITLE
feature(#1023): use digiwf-email lib to send emails

### LINKED ISSUE #1023: Task-Benachrichtigung enthält keine Zeilenumbrüche
**Describe the bug**

Bei einer individuelle Benachrichtigung für Aufgaben werden Zeilenumbrüche für die Mail nicht übernommen

**To Reproduce**

Siehe internes Ticket

**Expected behavior**

Zeilenumbrüche werden übernommen in der Mail

**Screenshots**
Siehe internes Ticket

**Additional context**
https://git.muenchen.de/digitalisierung/digiwf-support/-/issues/408

Nachträgliche Anmerkung: Das Ticket hat sich geändert siehe https://github.com/it-at-m/digiwf-core/issues/1023#issuecomment-1822300791

### PR DESCRIPTION
**Pull request by lmoesle**
### Description

Short description or comments

### Reference

Issues: closes #1023

### Check-List

- [x] All Acceptance criteria of user story are met
- [x] JUnit tests are written (60% CodeCov)
- [x] [Internal Review]([https://github.com/it-at-m/digiwf-core/blob/dev/CHANGELOG.md](https://confluence.muenchen.de/display/MPdZ/Review+-+DigiWF)) is maintained
- [x] Documentations [external](https://github.com/it-at-m/digiwf-core/tree/dev/docs) and [internal](https://wiki.muenchen.de/betriebshandbuch/index.php?title=DigiWF&sfr=betriebshandbuch) are completed
- [x] Smoketest successful (Manual E2E-Test depending on Change) 
- [x] No Branch waste left
- [x] [Board](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) is up-to-date
- [x] Openshift environments are prepared (Secrets, etc.) and release-issue is maintained

### CODE REVIEW

File: digiwf-libs/digiwf-email/digiwf-email-api/src/test/java/de/muenchen/oss/digiwf/email/DigiwfEmailApiImplTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    void sendMailWithAttachments() throws MessagingException {
+        final List<FileAttachment> fileAttachments = List.of(
+                new FileAttachment("Testanhang", new ByteArrayDataSource("FooBar".getBytes(), "text/plain"))
+        );
+        this.digiwfEmailApi.sendMailWithAttachments(this.receiver, this.subject, this.body, this.replyTo, fileAttachments);
+
+        final ArgumentCaptor<MimeMessage> messageArgumentCaptor = ArgumentCaptor.forClass(MimeMessage.class);
+        verify(this.javaMailSender).send(messageArgumentCaptor.capture());
+
+        Assertions.assertEquals(2, messageArgumentCaptor.getValue().getAllRecipients().length);
```
Comment (zambrovski) [inline digiwf-libs/digiwf-email/digiwf-email-api/src/test/java/de/muenchen/oss/digiwf/email/DigiwfEmailApiImplTest.java]: kein AssertJ?

File: digiwf-libs/digiwf-email/digiwf-email-api/src/test/java/de/muenchen/oss/digiwf/email/DigiwfEmailApiImplTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    @Test
+    void sendMailWithCCAndBCC() throws MessagingException {
+        this.digiwfEmailApi.sendMail(this.receiver, this.subject, this.body, this.replyTo, this.receiverCC, this.receiverBCC);
+
+        final ArgumentCaptor<MimeMessage> messageArgumentCaptor = ArgumentCaptor.forClass(MimeMessage.class);
+        verify(this.javaMailSender).send(messageArgumentCaptor.capture());
+
+        Assertions.assertEquals(4, messageArgumentCaptor.getValue().getAllRecipients().length);
```
Comment (zambrovski) [inline digiwf-libs/digiwf-email/digiwf-email-api/src/test/java/de/muenchen/oss/digiwf/email/DigiwfEmailApiImplTest.java]: kein AsserJ?

**Review by zambrovski — COMMENTED** [review]
Lass uns über die assertion library reden

**Review by StephanStrehlerCGI — APPROVED** [review]
sieht soweit gut für mich aus obwohl ich es jetzt nicht bis in die allerletzt tiefe durchgeschaut habe.

**Pull request merged**